### PR TITLE
Fix #108: Removes unnecessary horizontal & vertical scroll-bars from the home page

### LINKED ISSUE #108: Remove the scroll bar after “Who uses EvalAI”
### Observed Behaviour 
Scroll bar is visible under `Who uses EvalAI` on normal screen. The scroll bar is used for medium and small screen. Find attached screenshot.

### Expected Behaviour
The scroll bar should not be visible on normal screen.

@RishabhJain2018 

a

### PR DESCRIPTION
**Pull request by sanketbansal**
### Fix #108 

Link to live demo: https://pr-103-evalai.surge.sh

#### Changes proposed in this pull request:

- Removes the unnecessary scroller at y axis from the home page UI.
- Removes the scroll bar from x-axis below the "who uses Eval-Ai".

 

Screenshot (41)

### CODE REVIEW

File: src/app/components/nav/header-static/header-static.component.scss [hunk 1]
```diff
@@ -194,7 +194,7 @@ header {
     margin-top: 15px;
     margin-left: 0;
     &.at-home {
-      margin-left:0px;
+      margin-left:0;
```
Comment (Shekharrajak) [inline src/app/components/nav/header-static/header-static.component.scss]: What was the problem with `px`?

File: src/app/components/home/home.component.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 .home-container {
   overflow-x:hidden;
-  overflow-y:scroll;
+  //overflow-y:scroll;
```
Comment (Shekharrajak) [inline src/app/components/home/home.component.scss]: Please don't comment and commit your code changes. It should be either present or removed (if there is any specific reason for the code comment, then it should be written properly).

File: package.json [hunk 3]
```diff
@@ -25,13 +25,13 @@
...
     "core-js": "^2.4.1",
-    "ngx-twitter-timeline": "^0.1.4",
+    "ngx-twitter-timeline": "0.1.4",
     "normalize.css": "^8.0.0",
     "rxjs": "^5.5.6",
     "zone.js": "^0.8.19"
   },
   "devDependencies": {
-    "@angular/cli": "1.6.5",
+    "@angular/cli": "^1.6.5",
```
Comment (Shekharrajak) [inline package.json]: Any reason?

**Review by Shekharrajak — COMMENTED** [review]
Can you please attach working GIF in the PR description ?

File: src/app/components/nav/header-static/header-static.component.scss (hunk 1 above)
Comment (sanketbansal) [reply src/app/components/nav/header-static/header-static.component.scss]: It is not a error but a CSS linting problem.

File: src/app/components/home/home.component.scss (hunk 2 above)
Comment (sanketbansal) [reply src/app/components/home/home.component.scss]: okay, I will keep in mind.  This the issue due to which the main is using y-axis scroller by default.
Even when there is no overflow.

File: package.json (hunk 3 above)
Comment (sanketbansal) [reply package.json]: so, that if a higher version of angular 1.x.x can be used.

**Comment by sanketbansal** [comment]
fix #108

**Comment by sanketbansal** [comment]
Screenshot (42)

### Removed inner y-axis scroll bar and x-axis scroll bar below "who uses eval-Ai"

Screenshot (41)

File: src/app/components/nav/header-static/header-static.component.scss (hunk 1 above)
Comment (gautamjajoo) [reply src/app/components/nav/header-static/header-static.component.scss]: There is no difference bwtween `0` and `0px`, both work the same. It is only meant for better readability.

**Review by gautamjajoo — COMMENTED** [review]
@sanketbansal Could you remove all the changes done apart from solving this issue. There is already a [PR](https://github.com/Cloud-CV/EvalAI-ngx/pull/96) for it.

**Comment by sanketbansal** [comment]
Have removed all changes done apart from resolving issue.

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**